### ISSUE TITLE
Proper user and developer documentation needs to be provided

### ISSUE THREAD

**Issue by CapPow** [CapPow/pdDesk#8]
**Is your feature request related to a problem? Please describe.**
Reviewer comments noted the lack of user documentation associated with both collNotes and collBook.

**Describe the solution you'd like**
Part of the transition from a local solution to a public one needs to incorporate detailed documentation. 
 - Priority should be given to end user documentation in the form of a more informative [README.md](https://github.com/CapPow/collBook/blob/master/README.md). 
 - Help menu items in both collNotes and collBook should link to the [README.md](https://github.com/CapPow/collBook/blob/master/README.md)
 - The "what is this" Qt features should be populated for each UI element in collBook.
 - Developer documentation should also be populated in the form of structured doc strings in all appropriate functions and classes.

**Describe alternatives you've considered**
 - an intermediate solution may be to link a recorded presentation of talk prepared for conferences. 
 - a YouTube video demonstrating the use from field-to-database would be an excellent supplement to these features, but it cannot replace them.

**Comment by CapPow** [comment]
@j-h-m, 
Would you take a look over the new end user documentation added in c7ba284? If we're happy with this, we should integrate into the README.md.

Thanks!

**Comment by j-h-m** [comment]
Check the "collNotes (mobile) Initial Setup" section, the sentence of the last paragraph appears to be cut off, "It is also reccomended to." 

I would recommend only having info in the README for developers (limited to installation, building, and contributing) and instead using the Wiki (which I'm working on) or even a github io site for end users.

I am going to continue to add the content of the getting_started_documentation to the repo's [wiki](https://github.com/CapPow/collBook/wiki) if you like the idea. I think there is a way to create subsections and format the right-hand menu so we can have pages for developers, end users, and anyone in-between.

**Comment by CapPow** [comment]
@j-h-m,

Good catch on the cut off sentence! I agree, your organization method sounds better! Although we should have a link early in the README to the wiki to direct newcomers. 

Thanks!
